user/tests/wiring/watchdog/watchdog.cpp

```diff
@@ -47,7 +47,8 @@ test(WATCHDOG_00_setup_disconnect_power_off_ncp) {
     Cellular.off();
     waitFor(Cellular.isOff, 120000);
     assertTrue(Cellular.isOff());
-#elif Wiring_WiFi
+#endif
+#if Wiring_WiFi
```

Comment:
Not related to the bootloader, but just a subtle thing I noticed while testing wdog on msom. Just a convenient spot to include this change